runtime/compiler/p/codegen/J9UnresolvedDataSnippet.cpp

```diff
@@ -66,6 +67,13 @@ J9::Power::UnresolvedDataSnippet::UnresolvedDataSnippet(
    {
    }
 
+uint8_t *J9::Power::UnresolvedDataSnippet::getAddressOfDataReference()
+   {
+   if (self()->getDataReferenceInstruction())
+      return self()->getDataReferenceInstruction()->getBinaryEncoding();
```

Comment:
With the new expansion phase, will the binary encoding address be available at that time?

Reply:
The binary encoding address is not available during instruction expansion, since that runs before binary encoding. This method is later called when populating the snippet after binary encoding is already run, so the binary encoding address _is_ available at that time.